File: code/community/1301706051602616330/payment-decline-resolution.js
require('dotenv').config();
const { Deepgram } = require('@deepgram/sdk');

// Create a Deepgram SDK client
dgClient = new Deepgram(process.env.DEEPGRAM_API_KEY);

(async () => {
    const url = 'https://storage.googleapis.com/deepgram-demos/voicemail.mp3';
    const options = { language: 'en-US' };

    try {
        const response = await dgClient.transcription.preRecorded({ url }, options);
        console.log('Transcription:', response);
    } catch (err) {
        console.error('Error:', err);
    }
})();

File: code/community/1301706051602616330/payment-decline-resolution.go
package main

import (
	"fmt"
	"os"

	"github.com/deepgram-devs/deepgram-go"
)

func main() {
	apiKey := os.Getenv("DEEPGRAM_API_KEY")
	client := deepgram.NewClient(apiKey)

	url := "https://storage.googleapis.com/deepgram-demos/voicemail.mp3"

	options := deepgram.TranscriptionOptions{
		Language: "en-US",
	}

	transcription, err := client.TranscribeUrl(url, options)
	if err != nil {
		fmt.Println("Error:", err)
	}

	fmt.Printf("Transcription: %+v\n", transcription)
}